src/main/java/org/kitteh/irc/client/library/element/ISupportParameter.java

```diff
@@ -207,6 +207,31 @@
     }
 
     /**
+     * Represents support for EXTBAN.
+     */
+    interface ExtBan extends ISupportParameter {
+        /**
+         * Parameter name.
+         */
+        String NAME = "EXTBAN";
+
+        /**
+         * Gets the extban prefix.
+         *
+         * @return extban prefix if present
+         */
+        Optional<Character> getPrefix();
+
+        /**
+         * Gets the extbans.
+         *
+         * @return extbans
+         */
+        @Nonnull
+        List<ChannelMode> getExtBans();
```

Comment:
extbans don't really function as channel modes (they're in the banlist, for one) - Perhaps a new class should be created for this for extra separation, even if it might mostly align with a ChannelMode?